lib/ansible/plugins/callback/logstash.py

```diff
@@ -86,6 +87,7 @@ def __init__(self):
             self.hostname = socket.gethostname()
             self.session = str(uuid.uuid1())
             self.errors = 0
+        self.start_time = datetime.now()
```

Comment:
It would be better to use `datetime.utcnow()`to account for changes in daylight savings time that could happen during a job run.